WD-Firmware/comrv/comrv.c

```diff
@@ -18,21 +18,113 @@
 * @file   comrv.c
 * @author Ronen Haen
 * @date   21.06.2019
-* @brief  The file implements the COM-RV interfaces
+* @brief  The file implements the COM-RV engine
 * 
 */
+_Pragma("clang diagnostic ignored \"-Winline-asm\"")
```

Comment:
please add comment for why we are doing this

Reply:
done in da336bd